lms/djangoapps/ccx/tests/test_field_override_performance.py

```diff
@@ -234,24 +234,24 @@ class TestFieldOverrideMongoPerformance(FieldOverridePerformanceTestCase):
         #     # of mongo queries,
         #     # of xblocks
         # )
-        ('no_overrides', 1, True, False): (35, 0, 6, 1),
-        ('no_overrides', 2, True, False): (41, 0, 6, 1),
-        ('no_overrides', 3, True, False): (51, 0, 6, 1),
-        ('ccx', 1, True, False): (35, 0, 6, 1),
-        ('ccx', 2, True, False): (41, 0, 6, 1),
-        ('ccx', 3, True, False): (51, 0, 6, 1),
-        ('ccx', 1, True, True): (48, 0, 6, 1),
-        ('ccx', 2, True, True): (41, 0, 6, 1),
-        ('ccx', 3, True, True): (51, 0, 6, 1),
-        ('no_overrides', 1, False, False): (35, 0, 6, 1),
-        ('no_overrides', 2, False, False): (41, 0, 6, 1),
-        ('no_overrides', 3, False, False): (51, 0, 6, 1),
-        ('ccx', 1, False, False): (35, 0, 6, 1),
-        ('ccx', 2, False, False): (41, 0, 6, 1),
-        ('ccx', 3, False, False): (51, 0, 6, 1),
-        ('ccx', 1, False, True): (48, 0, 6, 1),
-        ('ccx', 2, False, True): (41, 0, 6, 1),
-        ('ccx', 3, False, True): (51, 0, 6, 1),
+        ('no_overrides', 1, True, False): (34, 0, 6, 1),
+        ('no_overrides', 2, True, False): (40, 0, 6, 1),
+        ('no_overrides', 3, True, False): (50, 0, 6, 1),
+        ('ccx', 1, True, False): (34, 0, 6, 1),
+        ('ccx', 2, True, False): (40, 0, 6, 1),
+        ('ccx', 3, True, False): (50, 0, 6, 1),
+        ('ccx', 1, True, True): (47, 0, 6, 1),
+        ('ccx', 2, True, True): (40, 0, 6, 1),
+        ('ccx', 3, True, True): (50, 0, 6, 1),
+        ('no_overrides', 1, False, False): (34, 0, 6, 1),
+        ('no_overrides', 2, False, False): (40, 0, 6, 1),
+        ('no_overrides', 3, False, False): (50, 0, 6, 1),
+        ('ccx', 1, False, False): (34, 0, 6, 1),
+        ('ccx', 2, False, False): (40, 0, 6, 1),
+        ('ccx', 3, False, False): (50, 0, 6, 1),
+        ('ccx', 1, False, True): (47, 0, 6, 1),
+        ('ccx', 2, False, True): (40, 0, 6, 1),
+        ('ccx', 3, False, True): (50, 0, 6, 1),
```

Comment:
These all went down by one due to an earlier commit (by @schenedx, if I recall correctly)